### DISCUSSION TITLE
table: Search (CTRL+F) refers only to the visible area

### DISCUSSION THREAD

**Discussion by pfischer64** [grafana/grafana#57069] [Dashboards, Panels & Visualizations]
**What would you like to be added:** 
In earlier Grafana versions it was possible to search the complete content of a table with CTRL+F.  Currently, only the visible part of the table is searchable using  CTRL+F. So, is there a possibility to extend the search to the whole table? The Column based filtering is not really helpful as it only considers only one column.

**Why is this needed:** 
If a table has more rows than can be displayed in the area, the search will not work because "Serach" refers only to the visible area.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
